Source/src/WixSharp.Samples/Wix# Samples/Bootstrapper/NsisBootstrapper/setup.cs

```diff
@@ -139,6 +142,18 @@ public static void BuildScriptSampleWithOSValidationWin7(string msiFile, string
         bootstrapper.Build();
     }
 
+    public static void BuildScriptWithCompressor(string msiFile)
+    {
+        var bootstrapper = new NsisBootstrapper
+        {
+            Primary = {FileName = msiFile},
+            OutputFile = "BuildScriptWithCompressor.exe"
+        };
+
+        bootstrapper.Compressor = new Compressor(false, true, Compressor.CompressionMethod.Bzip2);
```

Comment:
false, true is not very informative, have you considered using Flags for this purpose ? Something like 
new Compressor(Method.Bzip2, CompressionOptions.SOLID | CompresionOptions.FINAL)

Reply:
Will change to Flags